pkgs/applications/audio/geonkick/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitLab, cmake, pkgconfig, redkite, libsndfile, rapidjson, libjack2, lv2, libX11, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "geonkick";
+  version = "2.2.0";
+
+  src = fetchFromGitLab {
+    owner = "iurie-sw";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0nb0q4pysgxpsm0zn5xgmb9xg2573pl8vx3b7q8pn8hr4pxrh8y2";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  buildInputs = [ redkite libsndfile rapidjson libjack2 lv2 libX11 cairo ];
+
+  enableParallelBuilding = true;
```

Comment:
`enableParallelBuilding` is enabled by default in meson and cmake packages.

Reply:
Thanks, all done.